metals/src/main/scala/scala/meta/internal/metals/onTypeRangeFormatters/MultilineString.scala

```diff
@@ -44,10 +61,19 @@ class MultilineStringFormattingProvider(
     }
   }
 
-  private def isMultilineString(token: Token): Boolean = {
```

Comment:
Please, don't move the methods around. It's almost impossible to see what changed in this class.